core/src/main/java/gyro/core/validation/CollectionMinValidator.java

```diff
@@ -0,0 +1,22 @@
+package gyro.core.validation;
+
+import java.util.Collection;
+
+import gyro.core.resource.Diffable;
+
+public class CollectionMinValidator implements Validator<CollectionMin> {
+
+    @Override
+    public boolean isValid(Diffable diffable, CollectionMin annotation, Object value) {
+        if (value instanceof Collection) {
+            return ((Collection<?>) value).size() >= annotation.value();
+        }
+
+        return true;
+    }
+
+    @Override
+    public String getMessage(CollectionMin annotation) {
+        return String.format("Size of the collection must be greater than or equal to @|bold %s|@", annotation.value());
```

Comment:
You can use `%d` instead as we already know the type of value is integer.

Reply:
done.